tests/bluetooth/bsim_bt/bsim_test_mesh/overlay_low_lat.conf

```diff
@@ -1,4 +1,3 @@
-CONFIG_BT_EXT_ADV=n
```

Comment:
Why is extended advertising changed here?

Reply:
`CONFIG_BT_EXT_ADV` is default `n` and an explicit `n`  in `overlay_low_lat.conf` is inconsistent with tests not using this overlay.

Reply:
It is y by default for mesh: https://github.com/zephyrproject-rtos/zephyr/blob/928e5bbcf08ccfe75f264d407e16fb1c7e82846e/subsys/bluetooth/Kconfig.adv#L10

Reply:
@PavelVPV  Any reason why it is `n` for nRF51 (7367980) ? If it is only used for multiple advertising sets, then it should work on nRF51 too, did it fail on target?

Reply:
IIRC it didn't fit in flash or RAM.

Reply:
ok... I will go back to `CONFIG_BT_EXT_ADV=n` for nrf51 (low lat variant).  BTW, the `periodic_adv` sample builds, runs, is extended advertising and is discoverable on iPhone 13 Pro.